website/templates/project/project.mako

```diff
@@ -188,7 +188,9 @@
                 <span data-bind="css: icon"></span>
                 </p>
 
-                % if node['description'] or 'write' in user['permissions']:
+                % if not node['description']:
+                    <% pass %>
```

Comment:
Rather than having a conditional that just "passes", can you change the existing conditional so it gives the desired output?